solc/CommandLineInterface.cpp

```diff
@@ -738,7 +743,11 @@ void CommandLineInterface::createFile(string const& _fileName, string const& _da
 	ofstream outFile(pathName);
 	outFile << _data;
 	if (!outFile)
-		BOOST_THROW_EXCEPTION(FileError() << errinfo_comment("Could not write to file: " + pathName));
+	{
+		serr() << "Could not write to file \"" << pathName << "\"." << endl;
```

Comment:
It is not possible to test for this with the current testing infra.

However, before:
```
Exception during output generation: /Users/alex/Projects/solidity/solc/CommandLineInterface.cpp(748): Throw in function void solidity::frontend::CommandLineInterface::createFile(const std::__1::string &, const std::__1::string &)
Dynamic exception type: boost::wrapexcept<FileError>
std::exception::what: FileError
[solidity::util::tag_comment*] = Could not write to file: outdir/C.bin
```

After:
```
Could not write to file "outdir/C.bin".
```